README.md

```diff
@@ -3,8 +3,36 @@
 The Rancher Desktop guest agent runs in Rancher Desktop VMs providing helper
 services.  It currently has the following functionality:
 
+# When Privileged Service is enabled (admin installation of Rancher Desktop):
+
+```mermaid
+flowchart  LR;
+ subgraph Host["HOST"]
+ rd{"Rancher Desktop"}
+ ps(("Privileged Service"))
+ rd <-.-> ps
+ end
+ subgraph VM["WSL VM"]
+ rdagent(("Guest Agent"))
+ api(("moby/containerd \n events API"))
+ rdagent <-.->  api
+ end
+ ps  <---> |Vtunnel| rdagent
+```
+
+## moby port forwarding (WSL)
+
+Rancher Desktop Guest Agent subscribes to docker event API to monitor the newely created published ports. It will then forwards the newely published ports over a `AF_VSOCK` tunnel (Rancher Desktop's `vtunnel`) to [Rancher Desktop Privileged Service](https://github.com/rancher-sandbox/rancher-desktop/tree/main/src/go/privileged-service) that runs on the host machine.
```

Comment:
`newely` → `newly`. (This occurs multiple times, including in the containerd section.)